src/dir.targets

```diff
@@ -172,4 +172,27 @@
 
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
+
+  <!-- Generate coverage for individual projects. -->
+  <PropertyGroup>
+    <ProjectInput>$(MSBuildProjectName).dll.coverage.xml</ProjectInput>
```

Comment:
`<ProjectInput>$(TargetFileName).coverage.xml</ProjectInput>` will suffice? I will hold off pushing it as another commit until the other comment's issue is cleared up.